contracts/Peeranha.sol

```diff
@@ -49,9 +50,18 @@ contract Peeranha is IPeeranha, Initializable {
         SecurityLib.setupRole(userContext, SecurityLib.DEFAULT_ADMIN_ROLE, msg.sender);
     }
 
-    function getRatingToReward(address user, uint16 rewardPeriod) external view override returns(int32) {
-        RewardLib.PeriodRating storage userPeriod = RewardLib.getUserPeriodRating(userContext.userRewards, user, rewardPeriod);
-        return userPeriod.ratingToReward;
+    function getActiveInCommunity(address user, uint16 rewardPeriod) external override returns(uint32[] memory) {
```

Comment:
`getActiveInCommunity` -> `getActiveInCommunities`

Change this function to `view`

Reply:
in this function we change value
` userContext.userRatingCollection.communityRatingForUser[user].userRewards[rewardPeriod].isPaid = true;`